_data/changelogs/component-header.yml

```diff
@@ -2,6 +2,16 @@ title: Header
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Reordered header items so that search bar appears last in mobile header.
+    summaryAdditional: Now, the header is in compliance with WCAG criterion 2.4.3 (Focus order) in mobile view. 
+      Users that want to keep the search bar at the top of the menu will need to adjust their markup.
```

Comment:
**question:** How should we recommend users restore the previous order?

**If we suggest adjusting markup:** It will also affect their desktop view.

**If we suggest using CSS order:** It will no longer be compliant with WCAG `2.4.3 (Focus order)` criterion. 

It might be a good idea to add to a dev sync or office hours discussion to discuss with the team.

Reply:
> [!IMPORTANT]
> We should include whichever we decide in the `3.9.0` release notes